tutorials/build_system/gradle_guide.md

```diff
@@ -16,77 +16,78 @@ The plugin adds Gradle tasks for the `build.gradle` file that enable developing
 The [README](https://github.com/JetBrains/gradle-intellij-plugin/blob/master/README.md) file for the `gradle-intellij-plugin` project is the reference for configuring these tasks.
  
 When getting started, there are several items to note on the README page:
-* At the top of the page, the latest production version (e.g. `0.4.8`) of the IntelliJ IDEA Gradle plugin is listed.
+* At the top of the page, the [latest production version](https://github.com/jetbrains/gradle-intellij-plugin#the-latest-version) (e.g. `0.4.8`) of the IntelliJ IDEA Gradle plugin is listed.
 * Also at the top is the minimum version of Gradle required to support the IntelliJ IDEA Gradle plugin.
-* The table of extended Gradle [Tasks](https://github.com/JetBrains/gradle-intellij-plugin/blob/master/README.md#tasks) has a succinct description for each task added by the plugin.
-  This page will focus on the configuration and use four of those tasks:
-  * The [Setup DSL](https://github.com/JetBrains/gradle-intellij-plugin/blob/master/README.md#setup-dsl) - `intellij { ... }`.
-  * The [Running DSL](https://github.com/JetBrains/gradle-intellij-plugin/blob/master/README.md#running-dsl) - `runIde { ... }`
-  * The [Patching DSL](https://github.com/JetBrains/gradle-intellij-plugin/blob/master/README.md#patching-dsl) - `patchPluginXml { ... }`
-  * The [Publishing DSL](https://github.com/JetBrains/gradle-intellij-plugin/blob/master/README.md#publishing-dsl) - `publishPlugin { ... }`
-* Examples are always a helpful resource, and at the bottom of the page are links to [example](https://github.com/JetBrains/gradle-intellij-plugin/blob/master/README.md#examples) open source IntelliJ Platform plugin projects based on Gradle.
+* The table of extended Gradle [Tasks](https://github.com/jetbrains/gradle-intellij-plugin#tasks) has a succinct description for each task added by the plugin.
+  This documentation will focus on the configuration and use four of those tasks:
+  * [Setup DSL](https://github.com/jetbrains/gradle-intellij-plugin#setup-dsl) - `intellij { ... }`.
```

Comment:
why is jetbrains lowercased?

Reply:
Good catch. I do not know why. I thought I copied those URLs directly from the browser. In any case, I will fix them.